crates/cli-util/src/opts.rs

```diff
@@ -72,6 +76,24 @@ pub(crate) struct NetworkOpts {
     pub request_timeout: u64,
 }
 
+impl CommonClientConnectionOptions for NetworkOpts {
+    fn connect_timeout(&self) -> Duration {
+        Duration::from_millis(self.connect_timeout)
+    }
+
+    fn keep_alive_interval(&self) -> Duration {
+        Duration::from_secs(60)
+    }
+
+    fn keep_alive_timeout(&self) -> Duration {
+        Duration::from_millis(self.request_timeout)
+    }
```

Comment:
Isn't this a different meaning than before? Before `request_timeout` defined a timeout for the overall request.